### DISCUSSION TITLE
ARC on EKS Fargate

### DISCUSSION THREAD

**Discussion by dmitrijrub** [actions/actions-runner-controller#1672] [Ideas]
Hi, is it works time spending time trying to set up ARC with fargate as backend of EKS cluster?

**Comment by toast-gear** [comment]
https://github.com/actions-runner-controller/actions-runner-controller/issues/631 we've had reports it's possible but no one has provided a `show and tell` in dicussions for the specifics

**Reply by crackedupcorson** [reply]
I'm interested in seeing if it's doable, and will give it a try in a few weeks and report back.

For context, I'm hosting ARC on EKS currently. 

- We're using managed nodes, on EKS 1.22
- Controller  & Webhook Server running on a standard node
- Runner Pods running on dedicated CICD nodes using taint/tolerance and node-selectors.  
- Awaiting response from external teams and peers around possibility of Fargate (from a security perspective).
-  I only used RunnerDeployments in my last POC, which was on chart 0.15.0, but I'd like to use RunnerSets now that we're running the latest, for caching Docker images on PVs, but unsure if that's truly doable with Fargate (or sensible)

**Reply by dmitrijrub** [reply]
As i understand the biggest problem is Fargate does not support privileged containers

**Reply by pkordes** [reply]
Do you need privileged containers in your run dockerd WITHIN the runner vs a side-container?

**Reply by dmitrijrub** [reply]
yes

**Reply by joshua-reddish** [reply]
Any updates here? @crackedupcorson @dmitrijrub
